File: www/packages/com/thrift/test/service/impl/PersonServiceImpl.php
<?php
/**
 * Created by PhpStorm.
 * User: liuzh
 * Date: 14-2-11
 * Time: 下午3:11
 */

namespace com\thrift\test\service\impl;


use com\thrift\test\service\PersonServiceIf;

class PersonServiceImpl implements PersonServiceIf {

    public function getValue()
    {
        $val = array(
            'test'=>'it is test'
        );

        return json_encode($val);
    }

    public function getPerson(\com\thrift\test\bo\Person $person)
    {
        $person->name = $person->name.', hello';

        return $person;
    }
}

File: thrift/com/thrift/test/service/impl/PersonServiceImpl.py
#encoding=utf-8
#__author__ = 'liuzh'
from com.thrift.test.bo.ttypes import Person
from com.thrift.test.service import PersonService

class PersonServiceImpl:
    def getValue(self):
        return  'it is test for python'
    def getPerson(self, person):
        print(person)
        return person
